build/svgo.yml

```diff
@@ -15,6 +15,10 @@ js2svg:
   indent: 2
 
 plugins:
+  - addAttributesToSVGElement:
+      attributes:
+        - focusable: false
+        - data-image: icon
```

Comment:
Why do we need `data-image`?

Also, note that we'd need to run svgo with the updated config before merging this.

Reply:
sorry, cargo-culted that `data-image` part from the plugin's documentation, but you're quite right it's not needed. force-pushed an updated version without it.

when you say "we'd need to run svgo" ... where do you mean? against the files in docs? when i initially tried, it choked on the `{{ ... }}` etc liquid template bits.

Reply:
Yeah, it won't work for those where we have liquid code, but it'll work for the rest. I will take care of it and push here later.

Reply:
Actually I looked at it and it's OK as it is since svgo adds whitespace with this config.